models/stable_diffusion_v1_5.py

```diff
@@ -0,0 +1,87 @@
+import datetime
+import json
+import os
+import shutil
+import dtlpy as dl
+import torch
+from diffusers import StableDiffusionPipeline
+
+STREAM_URL = r"https://gate.dataloop.ai/api/v1/items/{}/stream"
+
+
+def create_folder(folder):
+    if os.path.exists(folder):
+        shutil.rmtree(folder, ignore_errors=True)
+    os.mkdir(folder)
+
+
+class HuggingAdapter:
+    def __init__(self, configuration):
+        self.model_name = configuration.get("model_name")
+
+        self.model = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5",
+                                                             torch_dtype=torch.float16)
+        self.model.to("cuda")
+        self.results_local_path = "stable_diffusion_v1_5_results"
+        create_folder(self.results_local_path)
+
+    def prepare_item_func(self, item: dl.Item):
+        dataset_id = item.dataset_id
+        buffer = json.load(item.download(save_locally=False))
+        prompts = buffer["prompts"]
+        ready_prompts = []
+        for prompt_key, prompt_content in prompts.items():
+            prompt_text = prompt_content[0]["value"]
```

Comment:
An example of the prompt would also be helpful here. I tried using a prompt in the same format as the one used for the LLMs and exemplified in the README file, but when running the preparation, it failed here because `prompt_content` does not have an index 0. It is a dictionary, and has no key 0 either. The input should follow the format used by the LLM adapters.